src/System.DirectoryServices/tests/System/DirectoryServices/DirectoryServicesTests.cs

```diff
@@ -303,6 +304,175 @@ public void TestDeleteTree()
             }
         }
 
+        [ConditionalFact(nameof(IsLdapConfigurationExist))]
+        public void TestDirectoryEntryNegativeCases()
+        {
+            // Test invalid server path
+            using (DirectoryEntry de = new DirectoryEntry("SomeWrongPath"))
```

Comment:
can you break these into separate tests? doesn't seem like they need to be in the same method.

Reply:
@danmosemsft could you please clarify why you think these are not different test cases and need to split into different methods? All these cases are testing the negative cases (the cases which throw exceptions) with the wrong usage.  I am seeing all these cases are related but I am open to split it if you can tell more details why you think so. Thanks.

Reply:
I have split it into smaller methods. please let me know if you have any other comment.